api/capi/src/tensor_filter_single.c

```diff
@@ -214,6 +216,7 @@ g_tensor_filter_single_start (GTensorFilterSingle * self)
     return FALSE;
 
   gst_tensor_filter_load_tensor_info (&self->priv);
+  self->allocate_in_invoke = gst_tensor_filter_allocate_in_invoke (priv);
```

Comment:
About first_invoke flag, how about using `priv->configured` flag in g_tensor_filter_single_start() function?
```
if (prop->input_configured && prop->output_configured)
  priv->configured = TRUE;

return priv->configured;
```
then, in g_tensor_filter_single_invoke() we can simply check the configured flag.
```
/** start if not already started */
if (priv->configured == FALSE) {
  if (!g_tensor_filter_single_start (self)) {
    return FALSE;
}
```

Reply:
That looks better. Updated.